spec/views/events/show.html.erb_spec.rb

```diff
@@ -54,17 +54,8 @@
     expect(rendered).to have_link(t(:details, scope: 'events.applicants_overview'))
   end
 
-  it "displays print applications button" do
-    render
-    expect(rendered).to have_link(t(:print_all, scope: 'events.applicants_overview'))
-  end
-
-  it "displays print badges button" do
-    render
-    expect(rendered).to have_link(t(:print_button_label, scope: 'events.badges'))
-  end
-
   it "should not display accept-all-button for non-organizers" do
+    @event = assign(:event, FactoryGirl.create(:event, :in_selection_phase))
     @event.max_participants = Float::INFINITY
     [:coach, :student].each do | each |
```

Comment:
I don't know why that is there, should I change it to pupil? (looks like this was somehow not renamed, I think in the very beginning there was a student)